net/sched/act_skbedit.c

```diff
@@ -115,6 +142,8 @@ static int tcf_skbedit_init(struct net *net, struct nlattr *nla,
 	}
 
 	parm = nla_data(tb[TCA_SKBEDIT_PARMS]);
+	if (parm->flags & SKBEDIT_F_INHERITDSFIELD)
```

Comment:
Does this code compile? `struct tc_skbedit` does not have the field `flags` like `struct tc_skbmod` does. Add the field `flags` to `struct tc_skbedit`.